### PR TITLE
activity type min and max duration

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3325733&stafilter=NotComplete&fileview=grid

### CODE REVIEW

File: src/components/forms/event-type-form.js [hunk 1]
```diff
@@ -373,6 +373,31 @@ class EventTypeForm extends React.Component {
...
+                                id="min_duration"
+                                type="number"
+                                value={entity.min_duration}
+                                onChange={this.handleChange}
+                                className="form-control"
+                                error={hasErrors('min_duration', errors)}
+                            />
+                        </div>
+                        <div className="col-md-4">
+                            <label> {T.translate("edit_event_type.max_duration")}</label>
```
Comment (smarcet) [inline src/components/forms/event-type-form.js:390]: lets add an info button with a tooltip ( title ) that states ( its only applies to track chairs )

File: src/components/forms/event-type-form.js [hunk 2]
```diff
@@ -373,6 +373,31 @@ class EventTypeForm extends React.Component {
                             </div>
                         </div>
                     </div>
+                    <hr/>
+                    <div className="row form-group">
+                        <div className="col-md-4">
+                            <label> {T.translate("edit_event_type.min_duration")}</label>
```
Comment (smarcet) [inline src/components/forms/event-type-form.js:379]: lets add an info button with a tooltip ( title ) that states ( its only applies to track chairs )

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments
regards

File: src/components/forms/event-type-form.js (hunk 1 above)
Comment (romanetar) [reply src/components/forms/event-type-form.js]: done

File: src/components/forms/event-type-form.js (hunk 2 above)
Comment (romanetar) [reply src/components/forms/event-type-form.js]: done

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**